tests/v1/entrypoints/conftest.py

```diff
@@ -79,73 +79,83 @@ def unsupported_json_schema():
             "score": {
                 "type": "integer",
                 "minimum": 0,
-                "maximum": 100  # Numeric range
+                "maximum": 100,  # Numeric range
             },
             "grade": {
                 "type": "string",
-                "pattern": "^[A-D]$"  # Regex pattern
+                "pattern": "^[A-D]$",  # Regex pattern
             },
             "email": {
                 "type": "string",
-                "pattern": "^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\\.[a-zA-Z]{2,}$"
+                "pattern": "^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\\.[a-zA-Z]{2,}$",
             },
             "tags": {
                 "type": "array",
                 "items": {
                     "type": "string",
-                    "pattern":
-                    "^[a-z]{1,10}$"  # Combining length and pattern restrictions
-                }
-            }
+                    "pattern": "^[a-z]{1,10}$",
+                    # Combining length and pattern restrictions
+                    "minLength": 1,
+                    "maxLength": 10,
+                },
+            },
         },
-        "required": ["score", "grade", "email", "tags"]
+        "required": ["score", "grade", "email", "tags"],
     }
 
 
 @pytest.fixture
 def sample_definition_json_schema():
     return {
-        '$defs': {
-            'Step': {
-                'properties': {
-                    'explanation': {
-                        'title': 'Explanation',
-                        'type': 'string'
+        "$defs": {
+            "Step": {
+                "properties": {
+                    "explanation": {
+                        "title": "Explanation",
+                        "type": "string"
+                    },
+                    "output": {
+                        "title": "Output",
+                        "type": "string"
                     },
-                    'output': {
-                        'title': 'Output',
-                        'type': 'string'
-                    }
                 },
-                'required': ['explanation', 'output'],
-                'title': 'Step',
-                'type': 'object'
+                "required": ["explanation", "output"],
+                "title": "Step",
+                "type": "object",
```

Comment:
I would prefer that the PR did not contain unrelated formatting changes. That makes it a bit harder to review. The comment applies to a bunch of changes.